src/benchmark/operators/sort_benchmark.cpp

```diff
@@ -22,28 +31,197 @@ class SortBenchmark : public MicroBenchmarkBasicFixture {
     }
   }
 
+  void BM_SortSingleColumnSQL(benchmark::State& state) {
+    _clear_cache();
+
+    auto& storage_manager = Hyrise::get().storage_manager;
+    auto column_names = std::optional<std::vector<std::string>>(1);
+    column_names->push_back("col_1");
+    storage_manager.add_table("table_a",
+                              GenerateCustomTable(ChunkOffset{2'000}, size_t{40'000}, DataType::Int, column_names));
+
+    const std::string query = R"(
```

Comment:
TODO Reduce code duplication: Factor out a protected method to only require setting the `query` variable in both `BM_SortSingleColumnSQL` and `BM_SortMultiColumnSQL` and only pass it to the protected method, where everything else happens.